### PR TITLE
[AX] <table> should be taken out of <figure> and the <figcaption> should become the table’s <caption>

### PR DESCRIPTION
**Pull request by marinaaisa**
Bug/issue #30234820, if applicable: 

## Summary

Tables were contained within a `<figure>` element. However, there is no need for this and it was making the tables much less accessible. Tables should stand on their own with the previous `<figcaption>` as the table’s `<caption>`. 

## Dependencies

NA

## Testing

Use the provided fixture folder:
[manual-fixtures.zip](https://github.com/apple/swift-docc-render/files/7803705/manual-fixtures.zip)

Steps:
1. Run `VUE_APP_DEV_SERVER_PROXY=/path/to/manual-fixtures/ npm run serve`
2. Go to http://localhost:8080/documentation/framework/sloth
3. Assert that tables' markup is as described in the Summary.

## Checklist

- [x] Added tests
- [x] Ran `npm test`, and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

**Comment by marinaaisa** [comment]
@swift-ci please test

File: src/components/ContentNode/Caption.vue [hunk 1]
```diff
@@ -9,31 +9,40 @@
...
+      required: false,
+      default: () => 'caption',
+    },
   },
 };
 </script>
 
 <style scoped lang="scss">
-@import 'docc-render/styles/_core.scss';
+@import '@/styles/_core.scss';
```
Comment (dobromir-hristov) [inline src/components/ContentNode/Caption.vue:35]:
```suggestion
@import 'docc-render/styles/_core.scss';
```

**Review by dobromir-hristov — APPROVED** [review]
Gave it a quick test. Looks the same hehe. Good job.

**Comment by marinaaisa** [comment]
@swift-ci please test

**Comment by marinaaisa** [comment]
@swift-ci test

File: src/components/ContentNode/Caption.vue [hunk 2]
```diff
@@ -53,4 +58,8 @@ export default {
 /deep/ p {
   display: inline-block;
 }
+
+caption {
```
Comment (dobromir-hristov) [inline src/components/ContentNode/Caption.vue:62]: I guess this is less of a concern, but if `caption-side: bottom;` is applied, this styling will not work as expected. However I am not sure how to check that with pure CSS. MDN jus suggest using padding around the caption...

**Review by dobromir-hristov — APPROVED** [review]
This works alright from what I could test. Left a small comment.

**Comment by marinaaisa** [comment]
@swift-ci test

**Pull request merged**